templates/service-template.yml

```
@@ -325,6 +330,11 @@ parameters:
   description: A list of denied users that are not allowed to access the service. A user is identified by its username.
   value: "[]"
 
+- name: ACCEPTED_USERS
+  displayName: A list of accepted users given by their usernames
+  description: A list of accepted users that are allowed to access the service. A user is identified by its username.
+  value: "[]"
```

In https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/901a40bf0f71bfecb768fd8218218db059e04d85/pkg/acl/access_control_list_middleware.go#L45 we are performing the checks against the organisation id, we might want to change the name of this parameter to reflect that its actually an organisation not users?
```suggestion
- name: ACCEPTED_ORGANISATIONS
  displayName: A list of accepted users given by their usernames
  description: A list of accepted users that are allowed to access the service. A user is identified by its username.
  value: "[]"
```

---

Thanks for spotting this @machi1990